dlrs/main/classes/RollupService.cls

```diff
@@ -1661,8 +1661,21 @@ global with sharing class RollupService {
         masterRecords.set(outerIndex, masterRecords.get(indexOfMin));
         masterRecords.set(indexOfMin, temp);
       }
+
+      // Create DmlOptions instance
+
+      Database.DmlOptions dmlOptions = new Database.DmlOptions();
+
+      // Allows save even if duplicated are detected
+      dmlOptions.duplicateRuleHeader.allowSave = true;
+
+      // Bypass duplicate rules
+
+
       try {
         return Database.update(masterRecords, allOrNothing);
```

Comment:
@chininchu were you able to test this code in an environment? You've put it back the way it was and aren't including the DmlOptions anymore.

Reply:
Yes, I was able to test the code and it works. You should now see the DML options on my latest commit.